game-server/data/handlers/quest/reshanta/_24044ChangeTheFuture.java

```diff
@@ -96,7 +96,7 @@ public boolean onDialogEvent(QuestEnv env) {
 							}
 							return false;
 						case SETPRO5:
-							return defaultCloseDialog(env, 4, 5); // 5
+							return defaultCloseDialog(env, 4, 5, 188020000, 1, 0, 0); // 5
```

Comment:
According to Munin's quest dialog, the player still has to craft or buy this stone himself.